source/manual/rabbitmq.html.md

```diff
@@ -95,3 +95,8 @@ last public_updated_at) for the entered document type:
 ```
 bundle exec rake queue:preview_recent_message[<document_type>]
 ```
+
+## Connecting to your local RabbitMQ
+
+Your local RabbitMQ is running as long as your development VM is running. 
+To see the control panel go to `dev.gov.uk:15672`.
```

Comment:
Not a big deal, but you could make this into a link.

Reply:
👍